elide-annotations/src/main/java/com/yahoo/elide/annotation/ReadPermission.java

```diff
@@ -37,5 +38,12 @@
      *
      * @return the class [ ]
      */
+    @Deprecated
     Class<? extends InlineCheck>[] all() default {};
+
+    /**
+     * An expression of checks that will be parsed via ANTLR.
```

Comment:
A little example here would be very useful to someone writing their first check expression.